components/sup/Cargo.toml

```diff
@@ -15,12 +15,14 @@ path = "src/main.rs"
 doc = false
 
 [dependencies]
-bytes = "*"
+bytes = "0.5.0"
 actix-web = { version = "*", default-features = false, features = [ "rust-tls" ] }
 byteorder = "*"
 clap = { version = "*", features = [ "suggestions", "color", "unstable" ] }
 cpu-time = "*"
-futures = "*"
+futures = { version = "0.3.1" }
+# futures 0.1 is needed because actix-web has not yet upgraded to futures 0.3
```

Comment:
An issue for tracking this work would also be nice.

Reply:
Added issue [here](https://github.com/habitat-sh/habitat/issues/7271)